icc-x-api/icc-message-x-api.ts

```diff
@@ -561,6 +561,7 @@ export class IccMessageXApi extends iccMessageApi {
         ])
           .compact()
           .uniq()
+          .filter(err => err.indexOf("510119") < 0)
```

Comment:
@gpiroux I do not understand what this does

Reply:
remove the errors 510119 that says that the record 51 is in error due to an other record (ET50)